docs/source/user/interface.rst

```diff
@@ -148,10 +148,18 @@ shortcuts. You can find and customize the current list of keyboard shortcuts by
 selecting the Advanced Settings Editor item in the Settings menu, then selecting
 Keyboard Shortcuts in the Settings tab.
 
-.. image:: images/interface_keyboard_shortcuts.png
```

Comment:
If you are replacing this image, can you also remove it from the repository?

Reply:
Sure